src/seedu/addressbook/util/FilePathUtil.java

```diff
@@ -0,0 +1,50 @@
+package seedu.addressbook.util;
+
+import java.io.File;
+import java.io.IOException;
+
+/**
+ * This is a utility class for file path validation.
+ */
+public class FilePathUtil {
```

Comment:
Sure I understand. Am I right to say that I will put FilePathUtil as an inner class of Addressbook?